packages/components/src/color-palette/index.js

```diff
@@ -33,9 +36,9 @@ export default function ColorPalette( {
 				selectedIconProps={
 					value === color
 						? {
-								fill: tinycolor
-									.mostReadable( color, [ '#000', '#fff' ] )
-									.toHexString(),
+								fill: colord( color ).isLight()
+									? '#000'
+									: '#fff',
```

Comment:
I think this change caused the failing unit tests, but I couldn't spot any regression when testing manually.

Reply:
I tried a simplification but it end up failing on some edge cases, our tests were good because they caught that. In practical terms, it would be unnoticeable because it means using white instead of black in backgrounds where they are almost as readable no matter the option. But I corrected it and now the result should be exactly the same as before.